napari/components/_layer_slicer.py

```diff
@@ -115,9 +119,14 @@ def wait_until_idle(self, timeout: Optional[float] = None) -> None:
             )
 
     def slice_layers_async(
```

Comment:
Is this name misleading? This is the long term path for both sync and async slicing, right?

Reply:
Great point! I'm in favor of renaming this. We could use `slice_layers`, though since this class is already `LayerSlicer` that's a bit repetitious. How about something like `submit` to match the `Executor` API, since we also have `shutdown`?

Reply:
`submit` sounds good to me!

Reply:
Done!